pymc_marketing/mmm/plot.py

```diff
@@ -243,6 +244,83 @@ def _build_subplot_title(
             return ", ".join(title_parts)
         return fallback_title
 
+    def _align_y_axes(self, ax_left, ax_right) -> None:
```

Comment:
@TeemuSailynoja can you please add type hinds and complete docstrings to this function so that we can bring it to the finish line :) Thanks!